report/renderer/report-ui-features.js

```diff
@@ -70,7 +70,7 @@ export class ReportUIFeatures {
     }
 
     // Fireworks!
-    // To get fireworks you need 100 scores in all core categories, except PWA (because going the PWA route is discretionary).
+    // To get fireworks you need 100 scores in all core categories.
     const fireworksRequiredCategoryIds = ['performance', 'accessibility', 'best-practices', 'seo'];
```

Comment:
My bad, I thought the logic blocklisted PWA instead of allowlisting the other LH categories. In that case I think it's still valuable to verify that fireworks are emitted in the case that a plugin is not at 100.

So I guess we can keep the test from https://github.com/GoogleChrome/lighthouse/pull/15455#discussion_r1544907907 and change it to test when an extra plugin category is not at 100.